server/routes/refreshrsRoutes.js

```diff
@@ -14,6 +14,24 @@ router.get('/', async (req, res, next) => {
   }
 });
 
+// gets all refreshrs belonging to a teacher
+// gonna come back and add authentication middleware later
+router.get('/teachers/:teacherId', async (req, res, next) => {
+  try {
+    const { teacherId } = req.params;
+    const refreshrList = await db.getTeacherRefreshrs(teacherId);
+    console.log(refreshrList);
+    res.status(responseStatus.success).json(refreshrList);
+  } catch (err) {
+    console.log(err);
+    if (TypeError) {
+      next(responseStatus.notFound);
```

Comment:
Interesting - so `if (TypeError)` means if the information is the wrong data / type of data, log notFound error?

Reply:
Yes, it looks like there is going to be a notfound error if the information is of the wrong type (integer, string, any).